libs/sdk-ui-dashboard/tsconfig.json

```diff
@@ -11,6 +11,7 @@
         "resolveJsonModule": true,
         "esModuleInterop": true,
         "importHelpers": true,
+        "downlevelIteration": true,
```

Comment:
Why is this needed?

Reply:
the filterValidation uses for-of loop to go through ObjRefMap.values() (which is iterator same as is in general Map contract)